nilearn/datasets.py

```diff
@@ -764,13 +796,12 @@ def fetch_craddock_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
     See http://www.nitrc.org/projects/cluster_roi/ for more information
     on this parcellation.
     """
-
     if url is None:
         url = "ftp://www.nitrc.org/home/groups/cluster_roi/htdocs" \
               "/Parcellations/craddock_2011_parcellations.tar.gz"
```

Comment:
Hmmm maybe not. This [page](http://www.nitrc.org/projects/cluster_roi/) by the authors has Craddock **2011**: 

![craddock_2011](https://cloud.githubusercontent.com/assets/1680079/6662423/ac33ee6e-cbbb-11e4-984d-3ec9842357d9.png)